helpers/import.sh

```diff
@@ -62,8 +62,9 @@ Example of required Terraform configuration:
       client_instance_metadata = {
         enable-oslogin = "TRUE"
       }
-      enable_write         = true
-      manage_rules_enabled = false
+      enable_write                 = true
+      manage_rules_enabled         = false
+      manage_server_config_enabled = false
```

Comment:
So, there is a new change in the `forseti_conf_server.yaml`, in this release.  How will user pick this up if this is disabled during the upgrade?

https://github.com/forseti-security/forseti-security/commit/fdfc1f90e9ca6099040a9d89fb33dc8f07a5f881#diff-c086fc0634ee9757f0ca86cd4a3df527